rse/grant-applicants.rst

```diff
@@ -0,0 +1,144 @@
+For grant applicants
+====================
+
+.. warning::
+
+   Grant applicants, please don't rely on this yet.  It will be
+   finalized soon.
+
+   This page is currently (2020-03) a draft for feedback.  Please send
+   corrections to the person that send this page to you, or rse-group
+   at aalto.fi
+
+
+
+If you've decided you would like to use the :doc:`research software
+engineer <index>` services in your project for a long period, you
+might want to write it directly into your grant proposals.  First off,
+your research will be better, second it may make your proposal more
+likely to get funded.
```

Comment:
The first point is surely true, how about rephrasing: "you can assign porting of codes, optimization of them, creating data analysis pipelines and such for the RSEs, while concentrating the main project resources to the actual research question."
I am unsure about the second point - no data to support it.

Reply:
Updated in latest version.
I'm also not sure about the second part, there is a reason I said "may".  It's been slightly reworded now.